lib/native-client.js

```diff
@@ -106,6 +106,56 @@ class NativeClient {
   }
 
   /**
+   * Get the currentOp.
+   *
+   * @param {Integer} includeAll - if true also list currently idle operations in the result.
+   * @param {Function} callback - The callback.
+   */
+  currentOp(includeAll, callback) { // TODO: deal with db versions? && do we want operations arg?
```

Comment:
I think the ability to simply pass `true` for everything as you have implemented is fine.